client/src/components/homepage/Cta.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import {Row, Col, Image, Button} from "react-bootstrap";
+import HomeImg from "../../assets/pexels-anna-shvets-resizeMed.jpg";
+
+export const Cta = () => {
+    return (
+        <div>
+            <Row className="row-cta">
+                <Col xs={12} className="home-img">
+                    <Image src={HomeImg} className="home-img-mob" alt="Home image" />
+                </Col>
+                <Col xs={12} className="cta-section">
+                    <p class="cta-title">Nothing is as strong as the heart of a volunteer</p>
+                    <div className="cta">
+                        <p class="cta-quote">The strong of the community is reflected by the passionate action of its members. Together with <span>Volunteer xChange Box</span>, find a volunteer activity in your area or call for a help from your community.</p>
+                        <Button>Join VoxBox Now</Button>
```

Comment:
@oguzkarademir good job, it is amazing .. can you make this div width:100% and its child p tag width as %90 or%95. Otherwise text overflow and user can not read all text